src/gps/GPS.h

```diff
@@ -128,14 +131,14 @@ class GPS : private concurrency::OSThread
 
     void setNumSatellites(uint8_t n);
 
-  private:
     /// Prepare the GPS for the cpu entering deep or light sleep, expect to be gone for at least 100s of msecs
     /// always returns 0 to indicate okay to sleep
     int prepareSleep(void *unused);
 
     /// Prepare the GPS for the cpu entering deep sleep, expect to be gone for at least 100s of msecs
     /// always returns 0 to indicate okay to sleep
     int prepareDeepSleep(void *unused);
+  private:
```

Comment:
Hesitant here to move them out of private, can you somehow do that via friend class? That will expose these routines only where they're needed.